dev/.rat-excludes

```diff
@@ -46,7 +46,7 @@ jsonFormatter.min.js
 .*txt
 .*json
 .*data
```

Comment:
Does it also apply to above .txt, .json, .data?

Reply:
To be safe, yes. Those are also looks suspicious.